pages/docs/install/containers/docker.html

```diff
@@ -47,15 +47,15 @@
 ```bash
 docker run -d -p 4200:4200 -p 4300:4300
   -v <data-dir>:/data crate
-  crate -Des.config=/path/to/crate.yml
-        -Des.cluster.name=cluster
+  crate -Cconfig=/path/to/crate.yml
```

Comment:
`-Cconfig` became `-Cpath.conf` and points to a folder (containing the logging and general configuration

also the site binding thing needs to go into here as well (from above)